src/dfx/src/lib/dist.rs

```diff
@@ -9,12 +9,11 @@ use indicatif::{ProgressBar, ProgressDrawTarget};
 use semver::Version;
 use std::fs;
 use std::io::Write;
-use std::path::Path;
 use tar::Archive;
 
 pub static DEFAULT_RELEASE_ROOT: &str = "https://sdk.dfinity.org";
-pub static CACHE_ROOT: &str = ".cache/dfinity/versions/";
-pub static DOWNLOADS_DIR: &str = ".cache/dfinity/downloads/";
+pub static CACHE_ROOT: &str = "dfinity/versions/";
```

Comment:
`CACHE_ROOT` feels like it's becoming a misleading name, but I don't really have a better idea